test/pytest/test_pointwiseconv.py

```diff
@@ -100,6 +105,7 @@ def test_pointwiseconv2d(chans, padds, strides, backend, io_type, strategy):
             kernel_initializer='normal',
             use_bias=False,
             data_format=chans,
+            name='pointwise2d',
```

Comment:
general comment, not specific to this line: why don't we do this for 2d as well?

Reply:
Yes, we can easily add 2d... working on it.